.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: release to quay.io
+on:
+  push:
+    tags: [v*]
```

Comment:
My understanding is that this build is going to be triggered by any branch as long as there is tag, should we try to limit this?

Reply:
I don't think we should limit, because tags are not limited to `main`, for instance `v0.1.0` was done on branch `release-4.10`.